.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Maintain Python packages
+  - package-ecosystem: "pip"
+    directory: "/"
+    target-branch: dev
+    commit-message:
+      prefix: "[Dependabot]"
+    labels:
+      - Dependencies
+    assignees:
+      - Paebbels
+    reviewers:
+      - Paebbels
+    schedule:
+      interval: "daily"    # Checks on Monday trough Friday.
+
+  # Maintain GitHub Action runners
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    target-branch: dev
+    commit-message:
+      prefix: "[Dependabot]"
+    labels:
+      - Dependencies
+    assignees:
```

Comment:
Do you want to be explicitly assigned for each PR? Otherwise, I'd suggest to remove all fields except `directory`, `target-branch` and `schedule`.

On the other hand, I suggest to add `open-pull-requests-limit` (see https://github.com/SymbiFlow/f4pga-arch-defs/commit/8a7fc2ff272222641fc5c887ca7d81a949c62bae#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R12-R16) which defaults to 5.

Reply:
We can remove the assignee, as it comes from a bot, but we should assign reviewers, because each submitted PR is a valid and complete PR. Thus it can be reviewed. Does GitHub support multiple reviewers? If so, I can add you too.

Currently, I see no need for an open PR limit. We can adjust later.

Reply:
Since we watch the repo, we will get notified anyway. I believe those features are more useful for projects with multiple maintainers, per are. Anyway, it's up to you. I think that lists are supported.

> Currently, I see no need for an open PR limit. We can adjust later.

The default is 5 I think. In order not to have a limit, you need to set it to e.g. 99..